### DISCUSSION TITLE
Typescript

### DISCUSSION THREAD

**Discussion by Lane-G-Smith** [Lane-G-Smith/AI-Personalities-React-App#17] [General]
@akpi816218 I'm not sure how typescript works, but it's messing with my IDE and causing syntax errors all over the place that I don't understand. Is there a way that I can just use normal JavaScript? example: Property 'div' does not exist on type 'JSX.IntrinsicElements'.

**Comment by ackledotdev** [comment]
Oh feel free to rename everything to `.js` and clean up anything that doesn't seem like vanilla JS. That should fix things.

If you want to try fixes first:
- `npm i`
- make sure component names are capitalized

**Comment by ackledotdev** [comment]
Apparently a clean `npm install` should do the trick, but you might need to delete `node_modules` and install `@types/react` and add `import * as React from 'react';`
